migrations/20220816142946-enhance-activities-links.js

```diff
@@ -0,0 +1,298 @@
+'use strict';
+
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    // -- Create columns --
+    console.time('Creating columns');
+
+    await queryInterface.addColumn('Activities', 'OrderId', {
+      type: Sequelize.INTEGER,
+      references: { key: 'id', model: 'Orders' },
+      onDelete: 'SET NULL',
+      onUpdate: 'CASCADE',
+      allowNull: true,
+    });
+
+    await queryInterface.addColumn('Activities', 'FromCollectiveId', {
+      type: Sequelize.INTEGER,
+      references: { key: 'id', model: 'Collectives' },
+      onDelete: 'SET NULL',
+      onUpdate: 'CASCADE',
+      allowNull: true,
+    });
+
+    await queryInterface.addColumn('Activities', 'HostCollectiveId', {
+      type: Sequelize.INTEGER,
+      references: { key: 'id', model: 'Collectives' },
+      onDelete: 'SET NULL',
+      onUpdate: 'CASCADE',
+      allowNull: true,
+    });
+
+    console.timeEnd('Creating columns');
+
+    // -- Fill in columns from existing data --
```

Comment:
@Betree : Maybe we could also do this same fill in columns from existing data for `collective.created` activity type. Note that the `CollectiveId` is contained within the `data` field there.

Reply:
`collective.created` is covered in https://github.com/opencollective/opencollective-api/blob/53cda4a23a1b35c708d5ee18b9a7218b03eaf578/migrations/20220816142946-enhance-activities-links.js#L285

Reply:
My thinking is that, `data.host.id` only exists for `collective.apply` activity. It seems that for `collective.created` what we need to copy is `data.collective.id` (to `CollectiveId` column).